system/systemd/PRE_BUILD

```diff
@@ -29,17 +29,23 @@ if mountpoint -q /usr && ! module_installed dracut; then
   message "${PROBLEM_COLOR}WARNING: ${MESSAGE_COLOR}You appear to have an /usr partition mounted. If you continue this"
   message "installation your computer might NOT be able to boot properly. Installing dracut to get initrd support"
   message "will take care of the issue.${DEFAULT_COLOR}"
-  if ! query "Ignore this warning and continue installation?" n ; then
+  if ! query "Ignore this warning and continue installation?" n; then
     exit 1
   fi
 fi
 
-default_pre_build &&
-# Compat .pc files
-patch_it $SOURCE2 1 &&
+default_pre_build
+
+# systemd guys wrongly assume that on every linux
+# installation is a group called "lock" this isn't
+# the case in lunar, so we switch it to root.
+sedit "s:0775 root lock:0775 root root:" tmpfiles.d/legacy.conf &&
+
+# Needed to migrate from udev module
+if module_installed udev; then
```

Comment:
This is too early in the chain and "could" potentiella break a system with udev still on it. But maybe it's time that we just remove all udev / sysvinit related migration code from our modules. I doubt anyone is still running a lunar system without systemd in place.